src/main/java/org/openjdk/jextract/impl/ToplevelBuilder.java

```diff
@@ -43,48 +43,76 @@ class ToplevelBuilder implements OutputFactory.Builder {
     private static final int DECLS_PER_HEADER_CLASS = Integer.getInteger("jextract.decls.per.header", 1000);
     public static final String PREV_SUFFIX = "#{PREV_SUFFIX}";
     private static final String SUFFIX = "#{SUFFIX}";
+    private static String SHARED;
```

Comment:
I think this was supposed to be `final`?
```suggestion
    private final String shared;
```